### DISCUSSION TITLE
[Bug Report] Unable to replace performer's picture

### DISCUSSION THREAD

**Discussion by Mitchametz** [stashapp/stash#5378] [Q&A] [answered]
**Describe the bug**
Unable to replace performer's picture

**To Reproduce**
1. Put a picture on performer (from file)
2. Validate the picture
3. Try to replace the picture (from file)
4. Validate the replacement picture
5. See error: 
writing to filesystem: creating file "blobs/87/bf/87bf2a8b896da384fd14ccfa2b906470": open blobs/87/bf/87bf2a8b896da384fd14ccfa2b906470: permission denied

**Expected behavior**
Have the other picture

**Screenshots**
If applicable, add screenshots to help explain your problem please ensure that your screenshots are SFW or at least appropriately censored.

**Stash Version: (from Settings -> About):**
v0.27.1

**Desktop (please complete the following information):**
 - OS: Debian 12.7.0
 - Browser : Vivaldi 6.9.3447.54 & Firefox 131.0.2

**Smartphone (please complete the following information):**
 - Device: Pixel 7 Pro
 - OS: Android 14
 - Browser: Vivaldi 6.9.3447.54 & Chrome

**Additional context**
This bug was not present in the precedent version

**Comment by Mitchametz** [comment, marked as answer]
**Solution:**
1. Put picture from url instead of picture from file
2. Validate
3. Put the picture from file
4. Validate

**Comment by feederbox826** [comment]
> writing to filesystem: creating file "blobs/87/bf/87bf2a8b896da384fd14ccfa2b906470": open blobs/87/bf/87bf2a8b896da384fd14ccfa2b906470: permission denied

looks like the permissions for your stash aren't configured properly, likely under your docker install. `chmod 664 -R STASH_PATH/.stash/blobs` should do the trick, otherwise pop in discord and ping me (same name) and I can try to help you troubleshoot
